src/containers/Application.js

```diff
@@ -124,11 +165,14 @@ class Application extends Component {
 		else return (
 			<Provider store={this.Store}>
 				<div className="app">
-					<Header app={this}/>
+					<Header hasSidebar={this.props.hasSidebar} app={this}/>
```

Comment:
You can access `app.props.hasSidebar` from Header.

No need to pass `hasSidebar={this.props.hasSidebar}` there.

Reply:
Changed in last commits